public/locales/en/translation.json

```diff
@@ -522,7 +522,10 @@
         "disable": "Disable",
         "title-field": "Title",
         "content-field": "Popup content",
-        "no-source": "Please select a source before configuring the popup"
+        "no-source": "Please select a source before configuring the popup",
+        "error": {
```

Comment:
I think there is generic message somewhere

Reply:
I will check

Reply:
I created a `datalayer.form.error-required` translation key instead, used by all validators